mdx-layout-core/src/CoreLayout.tsx

```diff
@@ -1,8 +1,26 @@
 import type { Context } from '@divriots/studio-doc-compiler';
 import { PageGraph, buildGraph } from './PageGraph';
-import './CoreLayout.scss';
 import { NavBar } from './NavBar';
-import React, { ReactChildren, ReactChild } from 'react';
+import React, { ReactChildren, ReactChild, CSSProperties } from 'react';
+
+import 'https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/themes/prism-tomorrow.min.css';
```

Comment:
did you check if this will give you what you want in the production build? it won't work the same way as in dev, you can check it with CLI `link` command

Reply:
@m4dz please double check this before merging, this is the only reason I'm not feeling like approving this PR

Reply:
To me this is the only limitation with the *embedded* styles: we let users pick and import their own prism stylesheets for their documentation. I guess it's relevant enough, especially about the backward compatibility: the worst case scenario will be without a prism stylesheet imported, the codeblocks won't be highlighted, but it won't break anything else. Sounds good to me :)